modules/nodes-nodes-audit-policy.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-audit-config.adoc
+
+[id="configuring-audit-policy_{context}"]
+= Configuring the node audit log policy
+
+You can configure the audit log policy to use when logging requests that come to the API servers.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `APIServer` resource:
++
+[source,terminal]
+----
+$ oc edit apiserver cluster
+----
+
+. Update the `spec.audit.profile` field:
++
+[source,yaml]
+----
+  apiVersion: config.openshift.io/v1
+  kind: APIServer
+  metadata:
+  ...
+  spec:
+    audit:
+      profile: WriteRequestBodies <1>
+----
+<1> Set to `Default`, `WriteRequestBodies`, or `AllRequestBodies`. The default profile is `Default`.
+
+. Save the file to apply the changes.
+
+. Verify that a new revision of the Kubernetes API server Pods has rolled out. This will take several minutes.
++
+[source,terminal]
+----
+$ oc get kubeapiserver -o=jsonpath='{range .items[0].status.conditions[?(@.type=="NodeInstallerProgressing")]}{.reason}{"\n"}{.message}{"\n"}'
+----
++
+Review the `NodeInstallerProgressing` status condition for the Kubernetes API server to verify that all nodes are at the latest revision. The output shows `AllNodesAtLatestRevision` upon successful update:
++
+[source,terminal]
+----
+AllNodesAtLatestRevision
```

Comment:
Possible spot for `.Example output`?

Reply:
@adellape I actually did have it initially, but then removed it because it had a lead-in sentence. wdyt, should we use it even with a lead-in sentence?